graal-sdk/build.gradle

```diff
@@ -44,9 +44,18 @@ publishing {
                 // Details about the artifact being published
                 groupId = "com.r3.conclave"
                 artifactId = "graal-sdk"
-                version = "$graal_version-$conclave_version"
+                version = conclave_graal_version
```

Comment:
In the `allprojects` block, can you set `version` to this value. Then all the sub-modules will have this version, and then automatically the publication will use this value as well.